src/_locales/pl.config

```diff
@@ -1,28 +1,28 @@
 @extension_description
-Tryb ciemny dla wszystkich stron internetowych. Zadbaj o swój wzrok, używając ciemnego motywu do przeglądania w dzień i w nocy.
+Pozwala na korzystanie z trybu ciemnego na każdej stronie internetowej. Zadbaj o swój wzrok, używając ciemnego motywu do przeglądania w dzień i w nocy.
 
 @loading_please_wait
 Ładowanie, proszę czekać
 
 @on
-Włącz.
+Włącz
```

Comment:
Nie usuwałbym tej kropki, bo może ona oznaczać skrót od "Włączone"
zobacz komentarz do @off

I would not remove this point as it can be shorted version of "ON" in polish and without point it means TURN IT ON.

Reply:
Znalazłem to w nowym designie i faktycznie może być bez kropek. Robię resolved.